### DISCUSSION TITLE
[SOLVED] TT-RSS: RSS Guard 4.3.0 AppImage Issue

### DISCUSSION THREAD

**Discussion by jvaccaro** [martinrotter/rssguard#879] [Q&A]
Hello;
Have just set up the AppImage version of RSS Guard 4.3.0.
I am running  RSS Guard on my desktop machine:
arach 
---------- 
OS: Ubuntu 20.04.5 LTS x86_64 
Host: Precision 5720 AIO 
Kernel: 5.15.0-58-generic 
Packages: 2990 (dpkg), 22 (flatpak), 22 (snap) 
Shell: bash 5.0.17 
Resolution: 3840x2160, 2560x1440, 1920x1080 
DE: GNOME 
WM: Mutter 
WM Theme: Adwaita 
Theme: Yaru-dark [GTK2/3] 
Icons: Papirus-Dark [GTK2/3] 
Terminal: gnome-terminal 
CPU: Intel Xeon E3-1245 v6 (8) @ 4.100GHz 
GPU: AMD ATI Radeon Pro WX 4130/4150 
GPU: Intel HD Graphics P630 
Memory: 14599MiB / 64172MiB 

The gui appears and I have set up my This Old Reader credentials and successfully retrieved my feed list.

However, RSS Guard never downloads any articles.

When I run the AppImage from the command line, there are tons of this message appearing:
_Delaying scheduled feed auto-downloads and message state synchronization for one minute due to another running update._

I assume this is why no articles are downloaded. I recreated the SQLite database thinking maybe it was corrupted, but nothing changed.

Any pointers or references would be appreciated.

JohnV
jvaccaro@gmail.com
-----------------------------------------------------------------------------------------------------
_Democracy is the recurrent suspicion that more than half of the people are right more than half of the time._

**Comment by martinrotter** [comment]
post full application log

**Comment by yesnomaybeyes** [comment]
I think I'm having a similar/same issue. I added my TT-RSS server, feed list loads fine but it acts as if there's no articles in any of the feeds. I reopen the application - same thing. So I then try to add the TTRSS demo server from their site and get the same issue.

![image](https://user-images.githubusercontent.com/28198579/215097534-30b7fc5a-3d68-47a9-8f28-90cc5430c04b.png)


log: https://gist.github.com/yesnomaybeyes/92bc20047e7d772bba4a680282403735

**Reply by martinrotter** [reply]
What does ttrss settings dialog in rssguard looks like?

**Reply by yesnomaybeyes** [reply]
![image](https://user-images.githubusercontent.com/28198579/215169660-5af0490d-180c-44b9-9de2-c8164a3d781e.png)

**Comment by martinrotter** [comment]
OK guys I investigated the problem and found the culprit.

"intelligent synchronization" sub-feature requires additional TT-RSS [plugin](https://github.com/hrk/tt-rss-newsplus-plugin) to be installed. I completely forgot to mention this somewhere in settings page of tt-rss in RSS Guard and will fix that.

So either install the plugin if you can (it is available for example on ttrss.info) or uncheck "intelligent synchronization".

I will fix in RSS Guard - add warning and make intelligent as unchecked by default on TT-RSS.

**Reply by martinrotter** [reply]
Adding proper error:

![image](https://user-images.githubusercontent.com/1255302/215415927-d8478699-c634-4793-af39-5f67c29789e4.png)
